baselines/aws/aws_v3_v5_migration/main.tf

```diff
@@ -4,6 +4,12 @@ resource "turbot_smart_folder" "turbot_event_handlers_folder" {
   parent = "tmod:@turbot/turbot#/"
 }
 
+resource "turbot_policy_setting" "aws_approved_regions" {
+  resource = turbot_smart_folder.turbot_event_handlers_folder.id
+  type = "tmod:@turbot/aws#/policy/types/approvedRegionsDefault"
+  value = var.aws_approved_regions
+}
+
```

Comment:
Why are you doing this? 

Approved Regions [Default] takes values from `AWS > Account > Regions` by default. 

```graphql
{
  regions: policyValue(uri:"tmod:@turbot/aws#/policy/types/regionsDefault") {
    value
  }
}
```

Why are you filtering?